### DISCUSSION TITLE
How to scale Ponder graphQL?

### DISCUSSION THREAD

**Discussion by tris4500** [ponder-sh/ponder#1002] [Question] [answered]
Can we have one instance of Ponder to index onchain event, process and store to a share database. Then we have multi instances of Ponder public graphQL endpoint and they just read data from the shared database?

**Comment by wyze** [comment, marked as answer]
The steps for Railway are outlined here: https://ponder.sh/docs/production/horizontal-scaling

Reading that, you could do something similar and switch from `ponder start` to `ponder serve` on your other nodes.
